hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ScheduleFeature.java

```diff
@@ -60,26 +65,63 @@ public class ScheduleFeature {
     private NetworkTransactionResponse networkTransactionResponse;
     private ScheduleId scheduleId;
     private TransactionId scheduledTransactionId;
+    private Duration plusSecondsToExpire;
+    private String scheduleTxConsensusTimestamp;
+
+    @Given(
+            "I successfully schedule a HBAR transfer from treasury to {account} with expiration time {string} and wait for expiry {string}")
+    public void createNewHBarTransferSchedule(
+            AccountNameEnum accountName, String expirationTimeInSeconds, String waitForExpiry) {
+        Instant expirationTime;
+        if (expirationTimeInSeconds.equals("null")) {
+            expirationTime = null;
+        } else {
+            this.plusSecondsToExpire = DurationStyle.detectAndParse(expirationTimeInSeconds);
+            expirationTime = Instant.now().plus(plusSecondsToExpire);
+        }
 
-    @Given("I successfully schedule a treasury HBAR disbursement to {string}")
-    public void createNewHBarTransferSchedule(String accountName) {
         currentSignersCount = SIGNATORY_COUNT_OFFSET;
-        var recipient =
-                accountClient.getAccount(AccountClient.AccountNameEnum.valueOf(accountName)); // receiverSigRequired
+        var recipient = accountClient.getAccount(accountName);
         var scheduledTransaction = accountClient.getCryptoTransferTransaction(
                 accountClient.getTokenTreasuryAccount().getAccountId(),
                 recipient.getAccountId(),
                 Hbar.fromTinybars(DEFAULT_TINY_HBAR));
 
-        createNewSchedule(scheduledTransaction, null);
+        createNewSchedule(scheduledTransaction, expirationTime, Boolean.parseBoolean(waitForExpiry));
+    }
+
+    @Given("I wait for the schedule to expire")
```

Comment:
because the scheduled transaction will execute after expiration time if all signatures are collected, a better description would be
```suggestion
    @Given("I wait until the schedule's expiration time")
```

Reply:
done